policy/scan/benchmark/benchmark_test.go

```diff
@@ -0,0 +1,121 @@
+// Copyright (c) Mondoo, Inc.
+// SPDX-License-Identifier: BUSL-1.1
+
+package benchmark
+
+import (
+	"context"
+	"testing"
+
+	"github.com/rs/zerolog"
+	"github.com/rs/zerolog/log"
+
+	"github.com/stretchr/testify/require"
+	"go.mondoo.com/cnquery/v10"
+	"go.mondoo.com/cnquery/v10/mqlc"
+	"go.mondoo.com/cnquery/v10/providers-sdk/v1/inventory"
+	"go.mondoo.com/cnquery/v10/providers-sdk/v1/testutils"
+	"go.mondoo.com/cnspec/v10/policy"
+	"go.mondoo.com/cnspec/v10/policy/scan"
+)
+
+func init() {
+	log.Logger = log.Logger.Level(zerolog.Disabled)
+	zerolog.SetGlobalLevel(zerolog.Disabled)
+}
+
+func BenchmarkScan_SingleAsset(b *testing.B) {
+	ctx := context.Background()
+	runtime := testutils.Local()
+	conf := mqlc.NewConfig(runtime.Schema(), cnquery.DefaultFeatures)
+	job := &scan.Job{
+		Inventory: &inventory.Inventory{
+			Spec: &inventory.InventorySpec{
+				Assets: []*inventory.Asset{
+					{
+						Connections: []*inventory.Config{
+							{
+								Type: "k8s",
```

Comment:
Normally we do not have any provider installed by default. Do we not need to ensure the k8s provider is installed. If so we also would need to clean this up afterwards to not mess with other tests.

Reply:
Yeah this will fail after we merged https://github.com/mondoohq/cnspec/pull/1060

Reply:
I don't think it will fail. When we run a scan, the required provider is auto-installed if it's not present. That should be fine. Since the goal of this benchmark is not to test provider installation or uninstall, I don't think it's necessary to cleanup providers. Here we only care about the CPU and memory usage during a scan